datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -368,14 +368,45 @@ pub fn to_substrait_rel(
                 .iter()
                 .map(|e| substrait_sort_field(state, e, sort.input.schema(), extensions))
                 .collect::<Result<Vec<_>>>()?;
-            Ok(Box::new(Rel {
+
+            let sort_rel = Box::new(Rel {
                 rel_type: Some(RelType::Sort(Box::new(SortRel {
                     common: None,
                     input: Some(input),
                     sorts: sort_fields,
                     advanced_extension: None,
                 }))),
-            }))
+            });
+
+            match sort.fetch {
```

Comment:
👍 

Another pattern I have seen to avoid missing fields like this is 

```rust
let Sort { input, offset } = sort ;
...
```
That way the compiler will tell you when you have not handled some field and you have to explicitly list it out

Reply:
Making sure I follow:

You mean on line 364 destructuring the plan like 
```rust
LogicalPlan::Sort(Sort {expr, input, fetch}) => {...
```
I didn't see that pattern followed in that file for the other LogicalPlan inners.   I think its a good idea though

Reply:
Done